Source/WebCore/svg/SVGTextPathElement.cpp

```diff
@@ -161,6 +162,12 @@ void SVGTextPathElement::buildPendingResource()
         }
     } else if (RefPtr pathElement = dynamicDowncast<SVGPathElement>(*target.element))
         pathElement->addReferencingElement(*this);
+
+    CheckedPtr renderer = this->renderer();
+    if (!renderer)
+        return;
+
+    LegacyRenderSVGResource::markForLayoutAndParentResourceInvalidation(*renderer);
```

Comment:
while I normally prefer early returns maybe a more conciece version of it would work better
```
if (CheckedPtr renderer = this->renderer();)
    LegacyRenderSVGResource::markForLayoutAndParentResourceInvalidation(*renderer);
```
but I am fine either way really.

Reply:
I noticed early return pattern across our codebase in SVG (with smart pointer adoption keeping it intact), so I think I will stick with it. :-)